lib/web_console/request.rb

```diff
@@ -32,6 +35,11 @@ def acceptable_content_type?
       content_type.blank? || content_type.in?(acceptable_content_types)
     end
 
+    # Returns whether the request is acceptable
+    def acceptable_version?
```

Comment:
Can we name this method `acceptable?` as the comment above implies and move the XHR check here too? That way, in the middleware, you'll only have:

```ruby
unless request.acceptable?
  return respond_with_unacceptable_request
end
```